peps/pep-0007.rst

```diff
@@ -135,6 +135,22 @@ Code lay-out
           return 0; /* "Forgive" adding a __dict__ only */
       }
 
+* Vertically align line continuation characters in multi-line macros.
+
+* Macros intended to be used as a statement should use the
```

Comment:
Perhaps provide an example of how you expect statement-like macros to be used?

Reply:
See https://github.com/python/peps/pull/3516/files#r1394825018